src/main/java/com/iyzipay/model/subscription/SubscriptionCustomerOperation.java

```diff
@@ -0,0 +1,17 @@
+package com.iyzipay.model.subscription;
+
+import com.iyzipay.HttpClient;
+import com.iyzipay.IyzipayResource;
+import com.iyzipay.Options;
+
+public class SubscriptionCustomerOperation extends IyzipayResource {
+
+    public static SubscriptionCustomerOperation delete(String subscriptionCustomerReferenceCode, Options options) {
```

Comment:
delete için ayrı bir class'a ihtiyaç olmayabilir. SubscriptionCustomer üzerinde yazıp response'ı IyzipayResource'a bind edebiliriz.

Reply:
Burada böyle yaparsak coupling artmış olur o yüzden ayrı yapıldı.